README.md

```diff
@@ -80,100 +71,95 @@ Now we'll convert it using HTML5 Semantic Elements
 </div>
 ```
 
-Notice that in cases where the content within the element is not semantically related or we have the need to simply create a generic box such as the `wrapper`, we can still use divs as we please.
+Notice that in cases where the content within the element is not semantically related or we have the need to create a generic box such as the `wrapper`, we can still use `div` elements as we please.
+
+## Identify Specific HTML5 Semantic Elements
 
-### Some Semantic Elements Worth Knowing
+Now that we have so many semantic elements we can use in our HTML pages, let's cover the most common and useful ones.
 
-Next let's talk about the semantic meaning intended for each element so you have an idea of how they were meant to be used. As a developer there are no hard rules of how to configure these elements. We only have their intended meaning to work off of, so you should feel comfortable configuring them in a way that makes the most sense to you and best suits your layout needs. The following descriptions are as they are stated from Mozilla Developer Networks documentation.
+As a developer, keep in mind that while these elements are intended for certain content, there are no hard rules about how to configure them. You should feel comfortable configuring them in any way that makes the most sense to you and best suits your layout needs.
 
-#### Article
+The following descriptions come from the documentation at the Mozilla Developer Network.
+
+### Article
 
 `<article>...</article>`
```

Comment:
I'd add in a suggested article structure here, as they're talking about having each article element identified as being a child of the article element, and that might be pretty unclear to a newbie.